.pre-commit-config.yaml

```diff
@@ -55,5 +55,9 @@ repos:
     rev: v8.56.0
     hooks:
     -   id: eslint
+        additional_dependencies:
+        -   eslint@8.56.0
```

Comment:
is this really needed as a dependency? I think by default it's included no?

Reply:
I think this is needed as a dependency for the pre-commit eslint mirror hook to run, removing this results to it failing with error "Executable `eslint` not found".